ClinGen/clincoded
# Connects to #877. Connects to #873. Connects to #875. Bugfixes to nav and population tables

#### Connects to #877:

*Testing:*
1. Go into VCI
2. Do stuff
3. Clicking the Home icon or ClinGen banner to go back to Dashboard should work

#### Connects to #873:

Fixed 0 vs false check for displaying 0 display in ExAC table. VEP ExAC allele frequency function was returning null values sometimes, despite having allele count values. Now, if allele count values are retrieved the AF is calculated. The VEP ExAC AF is only used if the AF values are null.

*Testing:*
1. Use CA ID 15333 to get to VCI
2. Go to population tab and confirm that the ExAC table is showing 0s instead of --s (they should be 0s). Confirm that the AF column is populated.

![image](https://cloud.githubusercontent.com/assets/4326866/17527328/c48eea16-5e1f-11e6-9b6c-1d85b8a15a74.png)

#### Connects to #875:

*Testing:*
1. Go to VCI
2. Go to Population tab and confirm that the ExAC table is showing the 'Other' row at the bottom, above totals

![image](https://cloud.githubusercontent.com/assets/4326866/17499990/74fa02a4-5d86-11e6-8048-b61bc3bd25af.png)
